benchmarks/dynamo/common.py

```diff
@@ -325,23 +325,6 @@ def nothing(f):
     return f
 
 
-@functools.lru_cache(None)
-def patch_torch_manual_seed():
-    """Make torch manual seed deterministic. Helps with accuracy testing."""
-
-    def deterministic_torch_manual_seed(*args, **kwargs):
-        from torch._C import default_generator
-
-        seed = 1337
-        import torch.cuda
-
-        if not torch.cuda._is_in_bad_fork():
-            torch.cuda.manual_seed_all(seed)
-        return default_generator.manual_seed(seed)
-
-    torch.manual_seed = deterministic_torch_manual_seed
-
-
```

Comment:
Is the code block useless?

Reply:
Yes, That code seems due to historical reason. The torch.manual_seed already has this logic:

Reply:
Since this PR should not do big changes. This change is reverted.